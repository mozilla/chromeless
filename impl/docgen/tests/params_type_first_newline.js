/**
 * A function which contains parameters expressed
 * with type before param name.
 */

/**
 * a function which converts URLs to filenames.
 * @param {string} url
 * The url to convert
 */
let toFilename = exports.toFilename = function toFilename(url) {
}
