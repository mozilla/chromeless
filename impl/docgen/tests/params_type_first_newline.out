{
  "desc": "A function which contains parameters expressed\nwith type before param name.", 
  "filename": "tests/params_type_first_newline.js", 
  "functions": [
    {
      "desc": "a function which converts URLs to filenames.", 
      "name": "toFilename", 
      "params": [
        {
          "desc": "The url to convert", 
          "name": "url", 
          "type": "string"
        }
      ]
    }
  ], 
  "module": "params_type_first_newline"
}
